hibernate-core/src/test/java/org/hibernate/jpa/test/graphs/Student.java

```diff
@@ -31,7 +31,7 @@
 	@NamedEntityGraph(
 		name = "Student.Full",
 		attributeNodes = {
-			@NamedAttributeNode(value = "courses")
+			@NamedAttributeNode(value = Student_.COURSES_)
```

Comment:
Now, I understand why you needed a constant. I wonder if we should also append the underscore at the end.

Reply:
You want me to remove the underscore?

Reply:
I personally find it odd as well.

Reply:
I removed the underscore